spring-integration-mongodb/src/test/java/org/springframework/integration/mongodb/store/ConfigurableMongoDbMessageStore-CustomConverter.xml

```diff
@@ -17,6 +17,7 @@
 	<beans:bean id="abstractMessageStore" class="org.springframework.integration.mongodb.store.MongoDbChannelMessageStore"
 				abstract="true">
 		<beans:constructor-arg name="mongoDbFactory" ref="mongoDbFactory"/>
+		<beans:property name="createIndex" value="false" />
```

Comment:
Doesn’t this still work if we move it to concrete extension of this bean?

Reply:
Yes! I don't know why.

Reply:
OK. I'll look into this when you are done with other my concerns in the production classes.